src/components/shared/Button.tsx

```diff
@@ -14,8 +14,11 @@ interface StyledElement {
 }
 
 interface ButtonContainer {
```

Comment:
Try to rename `ButtonContainer` => `ButtonContainerType` or `ButtonContainerProps` since you are making component named `ButtonContainer` below.